pygsti/report/workspaceplots.py

```diff
@@ -3033,13 +3034,13 @@ def _create(self, dataset, target, maxlen, fixed_lists, scale):
 
         xs = list(range(svals.size))
         trace1 = go.Bar(
-            x=xs, y=list(svals.flatten()),
+            x=xs, y=list(svals.flat),
```

Comment:
the `flat` property for ndarrays returns an instance of a `flatiter` object, so I had a suspicion this might be a bit slower. TLDR: For small arrays (4x4) `flat` is about 40% faster. For larger arrays (16x16 and 64x64) `flat` is about 15% slower. May well not matter here (depends on the typical size of svals, which I don't know off hand), but worth keeping in mind.

Reply:
Man that's wild. 

I'll switch to ``.ravel()`` for consistency, since it'll always be at least as fast as ``.flatten()`` and should be faster.